vspace/src/main/java/edu/asu/diging/vspace/web/staff/StaffSearchSpaceController.java

```diff
@@ -33,6 +40,26 @@ public ResponseEntity<StaffSearchSpaceResults> searchInVspace(
         staffSearch.setSpaces(spaceList);
         return new ResponseEntity<StaffSearchSpaceResults>(staffSearch, HttpStatus.OK);
     }
+    
+    @RequestMapping("/staff/spaces/search")
+    public ResponseEntity<String> search(@RequestParam(value = "term", required = false) String search){
```

Comment:
frontend should make a call to `searchInVspace`. No reason to implement a second type of search.